src/commands/templates/discover.js

```diff
@@ -17,25 +17,74 @@ const inquirer = require('inquirer')
 const { TEMPLATE_PACKAGE_JSON_KEY, readPackageJson } = require('../../lib/npm-helper')
 const { getTemplates } = require('../../lib/template-registry-helper')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app-templates:templates:discover', { provider: 'debug' })
+const loadConfig = require('@adobe/aio-cli-lib-app-config')
+const env = require('@adobe/aio-lib-env')
+const LibConsoleCLI = require('@adobe/aio-cli-lib-console')
 
 class DiscoverCommand extends BaseCommand {
+  async run () {
```

Comment:
@AmyJZhao , did you change `run()` ? I don't see any changes. Can you please move it below `__install()`, where it was before, so we can see exactly if it was changed or not.